ykman-gui/qml/CustomButton.qml

```diff
@@ -56,5 +58,20 @@ Button {
             text: button.text
             font: button.font
         }
+        Image {
+            visible: iconSource !== null && iconOnRight
+            id: iconRight
+            Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
+            fillMode: Image.PreserveAspectFit
+            sourceSize.height: 16
+            sourceSize.width: 16
+            source: iconSource
+
+            ColorOverlay {
+                anchors.fill: iconRight
+                source: iconRight
+                color: foregroundColor
+            }
+        }
```

Comment:
It would be nice to eliminate this code duplication, but it doesn't seem worth the effort to break this `Image` out as a separate component just for this.